docs/application/native/guides/connectivity/stc.md

```diff
@@ -0,0 +1,227 @@
+# STC
+## Dependencies
+- Tizen 4.0 and higher
+
+STC means Smart Traffic Control. It provides extensible packet-level control services, including per-app data usage, total data quota, and background app's data saving. STC Library provides APIs fulfilling below mentioned features for application development.
+
+This feature is supported in mobile, tv and werable profile.
+
+The main features of the STC API include:
+
+- Fetch Data Usage For System
+
+  You can [fetch network data consumed by system](#fetch_data_usage_system).
+
+- Fetch Data Usage For Applications
+
+  You can [fetch network data consumed by applications](#fetch_data_usage_applications).
+
+
+> **Note**  
+> You can test the STC functionality on a target device only. The [emulator](../../../../org.tizen.studio/html/common_tools/emulator.htm) does not support this feature.
+
+## Prerequisites
+
+To enable your application to use the STC API:
+
+1. To use the [STC](../../../../org.tizen.native.mobile.apireference/group__CAPI__NETWORK__STC__MODULE.html) API, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/network.get</privilege>
+   </privileges>
+   ```
+
+2. To use the functions and data types of the STC API, include the <stc.h> header file in your application:
+
+   ```
+   #include <stc.h>
+   ```
+
+3. Call the `stc_initialize()` function to initialize library:
+
+    ```
+    stc_initialize();
+    ```
+
+4. Call the `stc_deinitialize()` function to deinitialize library before quiting:
+
+   ```
+   stc_deinitialize();
+   ```
+
+## Fetch Data Usage For System
+
+To fetch statistics about total network data consumed by system:
```

Comment:
Suggestion to use a different word for 'Fetch'. 
Suggested words: Collect, retrieve.

Reply:
ok